nautilus-extension/nautilus-gsconnect.py

```diff
@@ -148,25 +128,24 @@ def get_file_items(self, *args):
 
         # Only accept regular files
         for uri in files:
-            if uri.get_uri_scheme() != 'file' or uri.is_directory():
-                return ()
+            if uri.get_uri_scheme() != "file" or uri.is_directory():
+                return tuple()
 
         # Enumerate capable devices
-        devices = []
+        devices = list()
```

Comment:
These changes baffle me, especially considering they seem to go in the opposite direction from all the other changes. Find me a person with more than 10 hours' familiarity with Python code who doesn't know that `[]` creates an empty list.

Reply:
Your the one stating all the way down I've made things unclear.  But here you are running for the hills as using names instead of literal empty syntax isn't clear.